src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -359,6 +372,17 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
+		}
+
+        internal void InitializeNamespacesToExcludeFromLibrary()
+        {
+            if (!NamespacesToExcludeFromLibrarySpecified)
+            {
+                NamespacesToExcludeFromLibrary.Add(
+                    "ProtoGeometry.dll:Autodesk.DesignScript.Geometry.TSpline"
```

Comment:
Yes, this property will be added to the XML once we run this version of Dynamo. Please refer to the PR description for more details. Thanks!